cmd/version.go

```diff
@@ -7,17 +7,4 @@ package main
 // Its value is collected in build time. See `make build/vidx2pidx`.
 var version string
 
-// License is a hardcoded string specifying vidx2pidx's license.
-const License = `Copyright 2021 Linaro
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.`
+const CopyrightNotice = " (C) 2025 Arm Ltd. and Contributors"
```

Comment:
keep linaro in the list for the copyright.

Reply:
Have [Reverted copyright notice](https://github.com/Open-CMSIS-Pack/vidx2pidx/pull/30/commits/11dd791f3ade9877e852c0c4dfe2135bcfdf2097) to show 
`vidx2pidx version X.X.X (C) 2021 Linaro`
The details on the license info from the `--version` command are removed. As license info is available with the release binaries.